spacy/language.py

```diff
@@ -134,73 +135,44 @@ def default_ner_labels(cls):
         return {0: {'PER': True, 'LOC': True, 'ORG': True, 'MISC': True}}
 
     @classmethod
-    def default_data_dir(cls):
-        return path.join(path.dirname(__file__), 'data')
-
-    @classmethod
-    def default_vocab(cls, data_dir=None, get_lex_attr=None):
-        if data_dir is None:
-            data_dir = cls.default_data_dir()
+    def default_vocab(cls, package=None, get_lex_attr=None):
+        if package is None:
+            package = default_package()
```

Comment:
The `Language.default_data_dir()` thing was fiddly, but there's an advantage to it: when you subclass, you have a way to specify a new default. If we read the default from this `default_package()` function, the subclass can't change that.

It could be that we don't need this capability. But that's why I did it like that.